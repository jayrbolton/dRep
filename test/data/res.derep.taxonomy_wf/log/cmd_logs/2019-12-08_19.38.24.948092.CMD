/usr/local/bin/mash paste /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/ALL.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/chunk_all.msh
