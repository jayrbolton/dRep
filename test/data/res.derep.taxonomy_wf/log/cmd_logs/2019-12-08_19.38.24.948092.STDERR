Writing /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/ALL.msh...
